src/app/personside/dialogpanel/fortsettDialog/tidligereMeldinger/EnkeltMelding.tsx

```diff
@@ -21,7 +21,7 @@ const StyledTekstomrade = styled(Tekstomrade)`
     padding-top: 0;
 `;
 
-const StyledEkspanderbartpanelBasePure = styled(EkspanderbartpanelBasePure)`
```

Comment:
`EkspanderbartpanelBasePure` finnes ikke for oss å bruke så dette er endret til `Base`